src/vm/amd64/pinvokestubs.S

```diff
@@ -24,14 +24,6 @@ LEAF_ENTRY GenericPInvokeCalliHelper, _TEXT
         test            rax, rax
         jz              C_FUNC(GenericPInvokeCalliGenILStub)
 
-        //
-        // We need to distinguish between a MethodDesc* and an unmanaged target in PInvokeStubForHost().
-        // The way we do this is to shift the managed target to the left by one bit and then set the
-        // least significant bit to 1.  This works because MethodDesc* are always 8-byte aligned.
-        //
-        shl             PINVOKE_CALLI_TARGET_REGISTER, 1
-        or              PINVOKE_CALLI_TARGET_REGISTER, 1
-
```

Comment:
This should have been removed when FEATURE_INCLUDE_ALL_INTERFACES was removed in  1476776a